### ISSUE TITLE
Possibly suspicious line?

### ISSUE THREAD

**Issue by ZedThree** [C-bowman/tokamesh#6]
This line in `construction.build_central_mesh` pads the input boundary by a fixed amount:

https://github.com/C-bowman/tokamesh/blob/feb0d0cf6063d9b4da7d41a90ab5e4e95a853a61/tokamesh/construction.py#L347

This looks like maybe it's supposed to pad by the height of two triangles, i.e. `2 * sqrt(3) * resolution * 0.5`? I don't think this affects the validity of the function -- it looks like it would only shift the start of the resulting grid?

On a related note, the docstring for `padding_factor` in this function says:

```
    :param padding_factor: \
        A multiplicative factor which defines the minimum distance to the boundary
        such that ``min_distance = padding_factor*scale``. No vertices in the returned
        mesh will be closer to the boundary than ``min_distance``.
```

Should `scale` be `resolution`?

**Comment by C-bowman** [comment]
Hi Peter,

Yep that's for sure a bug - I guess it's managed to hide because I tend to make meshes where `resolution` is much less than 1, which leads to creating way more padding than is needed, but then it's trimmed off anyway. This could definitely cause issues for cases with `resolution` bigger than 1, especially for coarser meshes.

The reference to `scale` in the docs is also an error for sure - I switched the name of the kwarg from `scale` to `resolution` a while ago and I must have missed that reference.

I'll commit a fix for it after I've merged your unit test PR,

Cheers
